### PR TITLE
Allow max year configuration for expirationDate validation

### PR DESCRIPTION
**Pull request by wozaki**
I want to customize the maximum year of expiry for my service too.

### CODE REVIEW

File: test/unit/expiration-year.js [hunk 1]
```diff
@@ -14,93 +14,110 @@ function yearsFromNow(fromNow, digits) {
...
+    });
+
+    it('accepts maxElapsedYear', function () {
+      expect(expirationYear(yearsFromNow(20), 20)).to.deep.equal({isValid: true, isPotentiallyValid: true, isCurrentYear: false});
+    });
+
+    it('returns invalid if beyond maxElapsedYear', function () {
+      expect(expirationYear(yearsFromNow(21), 20)).to.deep.equal({isValid: false, isPotentiallyValid: false, isCurrentYear: false});
+    });
+  });
```
Comment (wozaki) [inline test/unit/expiration-year.js]: `describe('maxElapsedYear'` is the main test in `expiration-year.js`.
The other difference is only indents 🙏 
https://github.com/braintree/card-validator/pull/54/files?w=1 will make a more readable difference.

File: src/expiration-year.js [hunk 2]
```diff
@@ -10,9 +10,11 @@ function verification(isValid, isPotentiallyValid, isCurrentYear) {
   };
 }
 
-function expirationYear(value) {
+function expirationYear(value, maxElapsedYear) {
```
Comment (wozaki) [inline src/expiration-year.js]: I renamed maxYear to maxElapsedYear because maxYear might remind someone of the dominical year.

**Comment by crookedneighbor** [comment]
Sorry for the delay.

Thanks for doing this. Made some slight adjustments to the code and merged it in. This will go out in the next release.

**Comment by crookedneighbor** [comment]
This was released as part of 4.2.0

**Comment by wozaki** [comment]
Thank you for releasing ✨

**Pull request closed without merging**